packages/vuepress-theme-odyssey/styles/docskit/_utility-classes.scss

```diff
@@ -0,0 +1,16 @@
+.is-sample-animation-off {
```

Comment:
Curious why many files have the underscore prefix. Is that a style-thing?

Reply:
It's a common style rule to track partials. Typically only compilation sheets like `main.scss` aren't underscored.